samples/oauth_rails_vethealth/app/views/oauth_callbacks/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<h1 class="display-2">Callback</h1>
+<p class="lead">
+    Now with the user identity verified by <em>ID.me</em>, we will make a
+    post to the Authorization Server with our client id and secret in the
+    authorization header.
+    The response will include a token we can use to access the Health API.
+</p>
+<%= render partial: 'post_data' %>
+<h3>Successfully authenticated OAuth.</h3>
+<table class="table">
+    <thead>
+        <tr>
+            <th scope="col">Response Key</th>
+            <th scope="col">Value</th>
+        </tr>
+    </thead>
+    <tbody>
+        <% @oauth_callback.response_body.keys.each do |key| %>
+        <tr>
+            <td><%= key %></td>
+            <td><%= @oauth_callback.response_body[key] %></td>
+        </tr>
+        <% end %>
+    </tbody>
+</table>
+<% if @oauth_callback.authentication %>
+    <% if @oauth_callback.authentication.authentication_errors.empty? %>
+        <p>Authentication was saved with ID <code><%= @oauth_callback.authentication.id %></code>.
+        Use this <%= link_to 'authentication record', authentication_path(@oauth_callback.authentication.id) %> to set headers for requests to the API.</p>
+    <% else %>
+        <p>The session does not match the oauth response.  These are the issues:</p>
+        <ul>
+        <% @oauth_callback.authentication.session_errors.each do |error| %>
```

Comment:
This should be `authentication_errors` now, right?

Reply:
thanks!  good catch!